src/controllers/controller.line.js

```diff
@@ -281,10 +281,16 @@ module.exports = DatasetController.extend({
 		if (me.chart.options.elements.line.capBezierPoints) {
 			for (i = 0, ilen = points.length; i < ilen; ++i) {
 				model = points[i]._model;
-				model.controlPointPreviousX = capControlPoint(model.controlPointPreviousX, area.left, area.right);
-				model.controlPointPreviousY = capControlPoint(model.controlPointPreviousY, area.top, area.bottom);
-				model.controlPointNextX = capControlPoint(model.controlPointNextX, area.left, area.right);
-				model.controlPointNextY = capControlPoint(model.controlPointNextY, area.top, area.bottom);
+				if (helpers.canvas.isPointInArea(model, area)) {
```

Comment:
I would store `helpers.canvas.isPointInArea` in a variable to help minimization.